### PR TITLE
add ellipsis to org name in settings

### LINKED ISSUE #3061: [Bug] Adding a long organisation name does not get truncated on editing from the organisational setting page
Steps to reproduce:
1) Navigate to Home page 
2) Click on organisation dropdown 
3) Click on Rename of organisation
4) Add a long name to the organisation
5) Navigate to organisational Setting
and observe the organisation name 

Observation:
It is observed that the organisation name is not truncated 

Expectation:
It is expected that when the organisational name is long it needs to be truncated

Screenshot 2021-02-16 at 4 22 42 PM

### PR DESCRIPTION
**Pull request by kaushik94**
## Description
Truncates org name in org settings

Fixes #3061 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/constants/DefaultTheme.tsx [hunk 1]
```diff
@@ -50,6 +50,13 @@ export const hideScrollbar = css`
   }
 `;
 
+export const addEllipsisToTextBlock = css`
```
Comment (riodeuno) [inline app/client/src/constants/DefaultTheme.tsx:53]: Let's rename this to `truncateTextWithEllipsis` or `EllipsizeText`. First alternative is more informative.

**Review by riodeuno — CHANGES_REQUESTED** [review]
One minor change requested. Rest looks good.

File: app/client/src/constants/DefaultTheme.tsx (hunk 1 above)
Comment (kaushik94) [reply app/client/src/constants/DefaultTheme.tsx]: @riodeuno I made the requested changes as well as merged release into. Please take a look.

**Review by riodeuno — APPROVED** [review]
Nice PR!
- Modular code
- Includes Cypress tests
- Doesn't seem to require functional jest test

LGTM!

**Pull request merged**